### ISSUE TITLE
Archiving file to custom path

### ISSUE THREAD

**Issue by albertony** [jotta/jotta-cli-issues#67]
Is archive option `--remote` still supported?

When testing issue #45, it seems the archive option `--remote` is no longer supported in version 0.5.13131. Trying something like this:

```jotta-cli.exe archive D:\Backup\subfolder\filename.7z --remote Backup```

Results in error:

```Error: unknown flag: --remote```

Help text `jotta-cli.exe archive --help` does not mention it either:

```
Upload a single file to the jottacloud archive

Usage:
  jotta-cli archive [flags]

Examples:

 jotta-cli archive /path/to/file --share
 will upload that file and print a http link to stdout


Flags:
      --clipboard   only usable with --share. copies the shared url to your clipboard
  -h, --help        help for archive
      --nogui       don't animate the upload progress
      --share       generate a public http url for the file after upload

Global Flags:
      --host string   the ip address of the host jottad is running on (default "127.0.0.1")
      --port string   the port jottad is listening on (default "14443")
```

But [official documentation](https://docs.jottacloud.com/jottacloud-command-line-tool/using-the-command-line-tool/archiving-files-with-jottacloud-cli) still describes it:
```
You can also specify a path under Archive to save the file to.

Example:
jotta-cli archive 'path/to/file' --remote folder/subfolder

This will save the file to Archive/folder/subfolder/file
```

**Comment by Kimbsen** [comment]
Just tested this and you are correct. We accidentally removed that flag. 

It will be back in the next version.

**Comment by Kimbsen** [comment]
We just released version 0.5.13645 in which ```jotta-cli archive --remote``` is back :)
